clients/cobol-lsp-vscode-extension/src/services/JavaCheck.ts

```diff
@@ -14,14 +14,21 @@
 import * as cp from "child_process";
 import { SettingsService } from "./Settings";
 
-const versionPattern = new RegExp(
-  '(java|openjdk) (version)? ?"?((9|[0-9][0-9])|(1|9|[0-9][0-9]).(1|8|[0-9][0-9]).*).*',
-);
+export const SUPPORTED_JAVA_VERSION = 11;
```

Comment:
Java 8 support has been restored.
```suggestion
export const SUPPORTED_JAVA_VERSION = 8;
```